db/migrate/20201104192101_add_contact_attempts_to_patients.rb

```diff
@@ -0,0 +1,21 @@
+class AddContactAttemptsToPatients < ActiveRecord::Migration[6.0]
+  def up
+    add_column :patients, :contact_attempts, :integer, default: 0
+
+    # populate :contact_attempts
+    execute <<-SQL.squish
+      UPDATE patients
+      INNER JOIN (
+        SELECT patient_id, COUNT(*) AS contact_attempts
+        FROM histories
+        WHERE history_type = 'Contact Attempt'
```

Comment:
I think you could also filter the `Patients` we loop through here - they don't have any histories anyway.

Reply:
ah good idea

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/499/commits/05512ce84e6034ae4b459c3de3800daa8bfe3ef3